js/src/components/contact-information/store-address-card.scss

```diff
@@ -0,0 +1,22 @@
+.gla-store-address-card {
+	// We could've used `StoreAddressCard'`s selectors,
+	// but the elements are wrapped with `<FlexItem>`,
+	// which does not have accessible semantic selector.
+	// So we have to rely on `<AccountCard>`'s internal implementation details.
```

Comment:
💅 

What about adding `className` onto these components?

https://github.com/woocommerce/google-listings-and-ads/blob/4de904a98e44aa45d839fb86043975feba0aba6e/js/src/components/account-card/index.js#L78

https://github.com/woocommerce/google-listings-and-ads/blob/4de904a98e44aa45d839fb86043975feba0aba6e/js/src/components/account-card/index.js#L84

Reply:
I proposed that with a bit more structured & generic way in https://github.com/woocommerce/google-listings-and-ads/pull/1030